### PR TITLE
Organizations API

### PR DESCRIPTION
**Pull request by bufke**
## Description

Allow creation/updating/deleting of organizations via API.

## Notes

I left in the create on list hack for now - though it really only runs on list view now. It won't do anything silly on other api views.

### CODE REVIEW

File: kobo/apps/organizations/tests/test_organizations_api.py [hunk 1]
```diff
@@ -46,3 +49,16 @@ def test_api_returns_org_data(self):
...
+        data = {'name': 'edit'}
+        res = self.client.patch(self.url_detail, data)
+        self.assertContains(res, data['name'])
+
+        user = baker.make(User)
+        self.client.force_login(user)
+        org_user = self.organization.add_user(user=user)
+        with self.assertNumQueries(3):
+            res = self.client.patch(self.url_detail, data)
+        self.assertEqual(res.status_code, 403)
```
Comment (LMNTL) [inline kobo/apps/organizations/tests/test_organizations_api.py:64]: Small question: why do you use `assertNumQueries` for a request that will 403? I don't think it's wrong, but I'm curious. I'd expect enforcing a certain number of queries for a successful request, but not for one that (hopefully) isn't accessing resources.

**Review by LMNTL — COMMENTED** [review]
This is looking good! I just have a question about a test case.

File: kobo/apps/organizations/tests/test_organizations_api.py (hunk 1 above)
Comment (bufke) [reply kobo/apps/organizations/tests/test_organizations_api.py:64]: The real reason is that I use that to debug, getting a printout of sql statements. I probably just left it here by mistake. I agree that it should be moved to a success path instead.

Comment (bufke) [reply kobo/apps/organizations/tests/test_organizations_api.py]: OK rebased and better tests (including new list view test). Thanks for spotting this.

**Review by LMNTL — APPROVED** [review]

**Pull request merged**